cmd/trade.go

```diff
@@ -48,7 +51,8 @@ func hiddenFlag(flag string) {
 func logPanic() {
 	if r := recover(); r != nil {
 		st := debug.Stack()
-		log.Printf("PANIC!! recovered to log it in the file\npanic: %v\n\n%s\n", r, string(st))
+		l.Errorf("PANIC!! recovered to log it in the file\npanic: %v\n\n%s\n", r, string(st))
+		os.Exit(1)
```

Comment:
why exit here?

Reply:
I thought the PANIC was supposed to exit, but now that you mention it I don't know why I would have thought I needed to add the exit here.